web/src/test/java/io/fabric8/launcher/web/api/HealthResourceIT.java

```diff
@@ -43,6 +45,21 @@
 @RunWith(Arquillian.class)
 public class HealthResourceIT {
 
+    @ClassRule
+    public static GitServer gitServer = GitServer.bundlesFromDirectory("repos/boosters")
```

Comment:
Wouldn't be better if this code stood in a common superclass? Or perhaps the @Deployment is adding too much instead of just adding HealthResource?

Reply:
I am not sure if we gain a lot from such a re-use. Afterall it's few lines of code vs making a constraint by introducing some "common" superclass.

But overall yes - it's deploying the whole app which might not be what we want.